Makefile

```diff
@@ -18,11 +18,11 @@ THIS_FILE := $(lastword $(MAKEFILE_LIST))
 
 -include Makefile.overrides
 
-MAISTRA_VERSION        ?= 2.4.3
+MAISTRA_VERSION        ?= 2.4.4
 MAISTRA_BRANCH         ?= maistra-2.4
 ISTIO_VERSION          ?= 1.16.5
```

Comment:
Does this need to be bumped as well? Haven't we updated `maistra/istio` to 1.16.7?

Reply:
@luksa Hmmm. Yes we are using 1.16.7, but I dont think we updated it in this version. Would it break anything if I were to update it in this PR?

Reply:
Yeah, it looks like we forgot to set this to 1.16.7 in the maistra-2.4.2 release.